### ISSUE TITLE
the paramaters in args.py

### ISSUE THREAD

**Issue by henbucuoshanghai** [wizyoung/YOLOv3_TensorFlow#76]
piecewise params
pw_boundaries = [60, 80]  # epoch based boundaries
pw_values = [learning_rate_init, 3e-5, 1e-4]

what is mean??  how to change it???

**Comment by henbucuoshanghai** [comment]
EVAL: Recall: 0.0037, Precison: 0.0001, mAP: 0.0000
EVAL: loss: total: 7655.04, xy: 1.30, wh: 21.19, conf: 7626.28, class: 6.28

at first ,the loss is very big???7655?
